scripts/index.js

```diff
@@ -109,4 +109,20 @@ function callNTimes(x, hello) {
 
 callNTimes(5, 'hello world');
 
+// **************** Str Multiply ************
+
+str = 'who run the world? Girls!';
+
+function beyonce(str, x) {
+    let arr = [];
+    for (let i = 0; i <= x; i++) {
+        arr.push(str);
+        console.log(arr);
+    }
+    return arr;
```

Comment:
Great job returning from your function. This is a great example of to use with your map.forEach, and filter functions.